src/GSoft.AspNetCore.Authentication.ClientCredentialsGrant/GSoft.AspNetCore.Authentication.ClientCredentialsGrant.csproj

```
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.16" />
```

```suggestion
    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="6.0.0" />
```

I believe we should target a minimal version of the package and let - and update our `renovate.json` to ignore `Microsoft.AspNetCore.*` at some point (in another story).